node-process-argv/echo.js

```diff
@@ -0,0 +1,5 @@
+let str = [];
+for (let i = 2; i < process.argv.length; i++) {
+  str.push(process.argv[i]);
+}
+console.log(str.join(' '));
```

Comment:
Overkill is underrated! You only needed `console.log(process.argv[2])`. But this is a more thorough solution.